lib/ProductOpener/Ingredients.pm

```diff
@@ -3472,11 +3472,42 @@ sub get_missing_ecobalyse_ids ($ingredients_ref) {
 
 		# List of suffixes
 		my @suffixes = ();
+		# If the ingredient is both organic and French...
+		if ((defined $ingredient_ref->{labels}) and 
+		    ($ingredient_ref->{labels} =~ /\ben:organic\b/) and 
+		    (defined $ingredient_ref->{origins}) and 
+		    (get_geographical_area($ingredient_ref->{origins}) eq "fr")) {
+
+		    push @suffixes, "_labels_en_organic_origins_en_france";
```

Comment:
If get_geographical_area() returns "fr", we need to push both suffixes: the "origins_en_france" one + the "origins_en_european_union_one" as we may not have a FR property in Ecobalyse, but we could have a EU one.